tutorials/hotspot.rst

```diff
@@ -101,47 +94,43 @@ following screenshot:
   :align: center
 
 This page communicates that the user can now use the internet
-provided by the *hotspot*, it also provides the following features:
+provided by the hotspot, it also provides the following features:
 
-- It shows a list of the user's sessions, including the start time,
+- It shows a list of the user’s sessions, including the start time,
```

Comment:
Let's not change this. I don't think these slant quotes are intentional, if they were we'd have to change the entire text.